Cannot use `zig cc` with `--target=wasm32-wasi` and `-D_WASI_EMULATED_SIGNAL`

### Zig Version

0.15.0-dev.386+2e35fdd03

### Steps to Reproduce and Observed Behavior

I might be holding things wrong...

I want to compile C code with `zig cc` with `target=wasm32-wasi` and `WASI_EMULATED_SIGNAL`; I get this error

```
 /zig/zig cc --target=wasm32-wasi -D_WASI_EMULATED_SIGNAL -I/some_software/out/include -D_WASI_EMULATED_SIGNAL
 -L/ghostscript-build/out/lib -lwasi-emulated-signal conftest.c  >&5
error: sub-compilation of libwasi-emulated-signal.a failed
    note: failed to check cache: '/zig/lib/libc/wasi/libc-top-half/musl/src/signal/psignal.c' file_hash FileNotFound
```

This is not nice. The file indeed doesn't exist.

I am using `-D_WASI_EMULATED_SIGNAL`, because without it, I get this error

```
/zig/lib/libc/include/wasm-wasi-musl/signal.h:2:2: error: "wasm lacks signal support; to enable minimal signal emulation, compile with -D_WASI_EMULATED_SIGNAL and link with -lwasi-emulated-signal"
```

Not sure what to do

### Expected Behavior

Everything builds

--- a/src/wasi_libc.zig
+++ b/src/wasi_libc.zig
@@ -1252,6 +1252,6 @@ const emulated_signal_bottom_half_src_files = &[_][]const u8{
 };
 
 const emulated_signal_top_half_src_files = &[_][]const u8{
-    "wasi/libc-top-half/musl/src/signal/psignal.c",
-    "wasi/libc-top-half/musl/src/string/strsignal.c",
+    "musl/src/signal/psignal.c",
+    "musl/src/string/strsignal.c",
 };

